OpenBSD + alpine3.7: testmod_network fails

```
 20/104 Test  #20: testmod_network ....................***Failed    0.01 sec
   ICONV   TESTS
====================

/root/Projekte/libelektra/src/plugins/network/testmod_network.c:27: error in test_addr: could not check ipv4 addr
/root/Projekte/libelektra/src/plugins/network/testmod_network.c:34: error in test_addr: could not check ipv4 addr
/root/Projekte/libelektra/src/plugins/network/testmod_network.c:41: error in test_addr: could not check ipv4 addr
/root/Projekte/libelektra/src/plugins/network/testmod_network.c:48: error in test_addr: could not check ipv4 addr

test_backendhelpers RESULTS: 8 test(s) done. 4 error(s).
```

```
 20/106 Test  #20: testmod_network ....................***Failed    0.02 sec
   ICONV   TESTS
====================

/home/mpranj/workspace/libelektra/src/plugins/network/testmod_network.c:27: error in test_addr: could not check ipv4 addr
/home/mpranj/workspace/libelektra/src/plugins/network/testmod_network.c:41: error in test_addr: could not check ipv4 addr
/home/mpranj/workspace/libelektra/src/plugins/network/testmod_network.c:62: error in test_addr: could not check ipv6 addr
/home/mpranj/workspace/libelektra/src/plugins/network/testmod_network.c:76: error in test_addr: could not do addr check (without specifying ipv4/ipv6)
/home/mpranj/workspace/libelektra/src/plugins/network/testmod_network.c:82: error in test_addr: could not do addr check (without specifying ipv4/ipv6)

test_backendhelpers RESULTS: 14 test(s) done. 5 error(s).
```

> Can you repost them please?

	75/158 Test  #75: testmod_network ..........................***Failed    0.02 sec
	NETWORK   TESTS
	===============

	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of ipv4 address <E2><80><9C>192.168.0.1<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of ipv4 address <E2><80><9C>192.168.1.1<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of ipv4 address <E2><80><9C>0.0.0.0<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of ipv6 address <E2><80><9C>2001:0db8:85a3:0000:0000:8a2e:0370:7334<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of ipv6 address <E2><80><9C>2001:0db8:85a3:0:0:8a2e:0370:7334<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of ipv6 address <E2><80><9C>2001:0db8:85a3::8a2e:0370:7334
	<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of ipv6 address <E2><80><9C>::1<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of ipv6 address <E2><80><9C>2001::7334<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of ipv6 address <E2><80><9C>::ffff:192.0.2.128<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of ipv6 address <E2><80><9C>::<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of IP address <E2><80><9C>::1<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of IP address <E2><80><9C>192.168.0.1<E2><80><9D> returned -1 instead of 1
	/root/libelektra/src/plugins/network/../ipaddr/test_ipaddr.h:24: error in testIP: validation of IP address <E2><80><9C>42.42.42.42<E2><80><9D> returned -1 instead of 1

> Does https://cmake.org/cmake/help/v3.0/variable/CMAKE_SYSTEM_NAME.html contain OpenBSD? 

    openbsd63# uname -s
    OpenBSD

Yes, it contains "OpenBSD". I verfied it with a short `CMakeLists.txt` file.

--- a/doc/news/_preparation_next_release.md
+++ b/doc/news/_preparation_next_release.md
@@ -127,6 +127,11 @@ Thanks to Michael Zronek and Vanessa Kos.
 - We fixed a memory leak in the [mINI plugin](https://libelektra.org/plugins/mini) by requiring the plugin
   [`ccode`](https://libelektra.org/plugins/ccode) instead of the “provider” `code`. *(René Schwaiger)*
 
+### network
+
+- Fixed an error in network plugin that prevented it from working on non glibc
+    platforms. *(Lukas Winkler)*
+
 ### Regex Dispatcher
 
 - The plugin [regexdispatcher](https://www.libelektra.org/plugins/regexdispatcher) has been added. It calculates regex representations for

--- a/src/plugins/network/network.c
+++ b/src/plugins/network/network.c
@@ -37,7 +37,7 @@ int elektraNetworkAddrInfo (Key * toCheck)
 	hints.ai_flags = AI_NUMERICHOST; /* Only accept numeric hosts */
 	hints.ai_protocol = 0;		 /* Any protocol */
 
-	s = getaddrinfo (keyString (toCheck), "", &hints, &result);
+	s = getaddrinfo (keyString (toCheck), NULL, &hints, &result);
 
 	if (s != 0)
 	{

